docs/buildah-bud.md

```diff
@@ -621,6 +640,8 @@ buildah bud --security-opt label=level:s0:c100,c200 --cgroup-parent /path/to/cgr
 
 buildah bud --volume /home/test:/myvol:ro,Z -t imageName .
 
+buildah bud -v /var/lib/dnf:/var/lib/dnf:O -t imageName .
+
```

Comment:
Will need to add this to Podman's build man page too.

Reply:
Yes I want to add full support for overlay mounts to podman